recipes/pexports/all/patches/0001-add-help.patch

```diff
@@ -0,0 +1,32 @@
+--- pexports.c
++++ pexports.c
```

Comment:
it seems be a feature patch, but doesn't break when missing

Reply:
Indeed. I added it to be able to test pexports on non-Windows systems.